metrics-operator/converter/sli_converter.md

```diff
@@ -0,0 +1,78 @@
+# SLI Converter
+
+SLI converter is a tool to convert the old `sli.yaml`
+[file format](https://github.com/keptn/spec/blob/master/service_level_indicator.md) into the new
+`AnalysisValueTemplate` [custom resource definition](../../docs/content/en/docs/crd-ref/metrics/v1alpha3/_index.md).
+The converter is part of `metrics-operator` image.
+
+## Usage
+
+The converter will convert a single `sli.yaml` file into multiple `AnalysisValueTemplate` resources.
+
+To run the converter, execute the following command:
+
+<!---x-release-please-start-version-->
```

Comment:
I think you need to adapt release please conf to let it know that the action has to watch the files

Reply:
adapted